.github/workflows/pre-main.yaml

```diff
@@ -383,6 +383,19 @@ jobs:
       - name: 'Test: Run Smoke Tests in a TNF container'
         run: TNF_LOG_LEVEL=${TNF_SMOKE_TESTS_LOG_LEVEL} ./run-tnf-container.sh ${{ env.TESTING_CMD_PARAMS }} -l "${SMOKE_TESTS_GINKGO_LABELS_FILTER}"
 
+      # Perform tests on collector
+      - name: Check out `Collector`
+        uses: actions/checkout@v3
+        with:
+          repository: shirmoran/collector
```

Comment:
i think u need to put the repo from test-network-function/collector

Reply:
It is set that way temporarily for testing. Will be changed.